src/debugger/gdb-stub.c

```diff
@@ -710,6 +766,8 @@ bool GDBStubListen(struct GDBStub* stub, int port, const struct Address* bindAdd
 		goto cleanup;
 	}
 
+	memset(stub->memoryMapXml, 0, GDB_STUB_MAX_LINE);
```

Comment:
This should be done in init, not here.

Reply:
I am not sure about this: moving it there means it's possible to open a connexion, then stop it, but keep the gdb server window open, then start a completely different game and then start a new connexion. This would mean the mapping could change, but GDB will be sent the previous one which is why I put it there because this code will only run when you open a sucessful connexion so we ensure the cached one is clear before we continue.

Reply:
You appear to be correct. You could get away with just setting the first byte to 0, but ultimately it won't matter either way.